include/functions.h

```diff
@@ -3858,7 +3764,7 @@ void func_801A1DB8(Vec3f* vec, u8 seqId, f32 arg2);
 // void func_801A1E0C(void);
 void func_801A1F00(u8 arg0, u16 seqId);
 void func_801A1F88(void);
-// void func_801A1FB4(void);
+void func_801A1FB4(u8 arg0, Vec3f* arg1, u16 bgmId, f32 arg3);
```

Comment:
```suggestion
void func_801A1FB4(u8 arg0, Vec3f* arg1, u16 seqId, f32 arg3);
```
Sequence Id is the convention used for this, since the bgm enum contains bgms, fanfares, sfxs, and nature ambiences.  `NA_BGM_` will probably be changed to `just `SEQ_` in the future, but that's still in discussion